jda/src/main/java/co/aikar/commands/JDACommandManager.java

```diff
@@ -47,6 +50,18 @@ public JDACommandManager(JDA jda, CommandConfig defaultConfig, CommandConfigProv
         this.configProvider = configProvider;
         this.completions = new JDACommandCompletions(this);
         this.logger = Logger.getLogger(this.getClass().getSimpleName());
+
+        if (jda.getAccountType() == AccountType.BOT) {
+            botOwner = jda.asBot().getApplicationInfo().complete().getOwner();
+        } else {
+            botOwner = jda.getSelfUser();
+        }
+
+        getCommandConditions().addCondition("owneronly", context -> {
+            if (context.getIssuer().getEvent().getAuthor() != botOwner) {
+                throw new ConditionFailedException("Must be bot owner");
```

Comment:
Lets use a bit more descriptive wording "Only the bot owner can use this command", and add a // TODO: MessageKey